schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js

```diff
@@ -30,7 +33,8 @@
         }
 
         function fail(r) {
-            vm.alert('danger', 'Failed: ' + r.statusText);
+            var translation = $translate('FAILED');
+            vm.alert('danger', translation + ': ' + r.statusText);
         }
```

Comment:
We could use error codes and only display those or use them to find the correct translation on the front end.